include/pm/runtime.h

```diff
@@ -0,0 +1,130 @@
+/*
```

Comment:
device_runtime.h ?

Reply:
I tend to prefer shorter names, but I'm open to change it, other opinions?

Reply:
I'd go for device_runtime as well.
Since we have PM and PM_DEVICE, making it clear this is targeting device may avoid confusions

Reply:
changed to `device_runtime.h`